### ISSUE TITLE
MODULE_NOT_FOUND when including a plugin

### ISSUE THREAD

**Issue by b0g3r** [ludanxer/catalog-graph#4]
Thank you for your awesome work, graph looks awesome! 🙌 

Unfortunately, I have an issue with connecting the plugin to a project. I added the plugin to `node_modules` and `config.js`, and when I started `vuepress dev targetDir --debug` I received `MODULE_NOT_FOUND` error.

Full debug output:
```
debug Plugins defined at @vuepress/internal-site-config [ [Function (anonymous)], 'vuepress-plugin-catalog-graph' ]
debug Error: Cannot find module '/Users/DBoger/PycharmProjects/tlroadmap/node_modules/vuepress-plugin-catalog-graph/src/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:318:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:700:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:963:27)
    at resolve (internal/modules/cjs/helpers.js:78:19)
    at resolveModule (/Users/DBoger/PycharmProjects/tlroadmap/node_modules/@vuepress/shared-utils/lib/moduleLoader.js:55:26)
    at Object.loadModule (/Users/DBoger/PycharmProjects/tlroadmap/node_modules/@vuepress/shared-utils/lib/moduleLoader.js:60:26)
    at ModuleResolver.resolveDepPackage (/Users/DBoger/PycharmProjects/tlroadmap/node_modules/@vuepress/shared-utils/lib/moduleResolver.js:140:34)
    at Object.tryChain [as default] (/Users/DBoger/PycharmProjects/tlroadmap/node_modules/@vuepress/shared-utils/lib/tryChain.js:13:24)
    at ModuleResolver.resolve (/Users/DBoger/PycharmProjects/tlroadmap/node_modules/@vuepress/shared-utils/lib/moduleResolver.js:84:44)
    at PluginAPI.normalizePlugin (/Users/DBoger/PycharmProjects/tlroadmap/node_modules/@vuepress/core/lib/node/plugin-api/index.js:121:39)
    at PluginAPI.use (/Users/DBoger/PycharmProjects/tlroadmap/node_modules/@vuepress/core/lib/node/plugin-api/index.js:89:23)
    at /Users/DBoger/PycharmProjects/tlroadmap/node_modules/@vuepress/core/lib/node/plugin-api/index.js:146:12
    at Array.forEach (<anonymous>)
    at PluginAPI.useByPluginsConfig (/Users/DBoger/PycharmProjects/tlroadmap/node_modules/@vuepress/core/lib/node/plugin-api/index.js:145:19)
    at PluginAPI.use (/Users/DBoger/PycharmProjects/tlroadmap/node_modules/@vuepress/core/lib/node/plugin-api/index.js:107:12)
    at App.applyUserPlugins (/Users/DBoger/PycharmProjects/tlroadmap/node_modules/@vuepress/core/lib/node/App.js:194:8) {
  code: 'MODULE_NOT_FOUND',
  path: '/Users/DBoger/PycharmProjects/tlroadmap/node_modules/vuepress-plugin-catalog-graph/package.json',
  requestPath: 'vuepress-plugin-catalog-graph'
}
warning An error was encountered in plugin "vuepress-plugin-catalog-graph"
```

**Comment by LeoHsiao1** [comment]
I also encountered this error when using `vuepress-plugin-catalog-graph@0.2.5`.
After analysis, I suspect that the author renamed the source code directory `src` to `lib`, but did not update the configuration.

So I changed the configuration manually:
1. Edit the file `node_modules/vuepress-plugin-catalog-graph/package.json`
2. Change `"main": "src/index.js"` to `"main": "lib/index.js"`.

Finally I managed to see this interesting directory diagram. But it seems that my site has too many directories and it takes 3 minutes to render.
